OptimizelySDK/OptlyConfig/OptimizelyConfig.cs

```diff
@@ -26,11 +27,14 @@ public class OptimizelyConfig
         public string SDKKey { get; private set; }
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
         public string EnvironmentKey { get; private set; }
+        public Entity.Event[] Events { get; private set; }
+        public OptimizelyAudience[] Audiences { get; private set; }
+        public Attribute[] Attributes { get; private set; }
```

Comment:
Is there a reason we're defaulting to use arrays vs just using ienumerable and allowing the consumer to determine how they process this?